packages/dds/tree/src/util/multimap.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export class Multimap<K, V> extends Map<K, V[]> {
```

Comment:
As is, this class doesn't really work like a multi map would be expected to work.

It "has" will return true for keys with no values. Calling `get` for a bunch of keys leaks memory. 

I feel like it should at least have some documentation other than its name to compensate for this.